src/test/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesServiceTest.kt

```diff
@@ -166,19 +167,18 @@ class ParticipantProjectSpeciesServiceTest : DatabaseTest(), RunsAsUser {
       val userId = currentUser().userId
       val now = clock.instant
 
-      assertEquals(
-          listOf(
-              SubmissionsRow(
-                  createdBy = userId,
-                  createdTime = now,
-                  deliverableId = deliverableId,
-                  feedback = "So far so good",
-                  id = submissionId,
-                  modifiedBy = userId,
-                  modifiedTime = now,
-                  projectId = projectId,
-                  submissionStatusId = SubmissionStatus.NotSubmitted)),
-          submissionsDao.fetchByDeliverableId(deliverableId))
+      assertTableEquals(
+          SubmissionsRecord(
+              createdBy = userId,
+              createdTime = now,
+              deliverableId = deliverableId,
+              feedback = "So far so good",
+              id = submissionId,
+              modifiedBy = userId,
+              modifiedTime = now,
+              projectId = projectId,
+              submissionStatusId = SubmissionStatus.NotSubmitted),
+          where = SUBMISSIONS.DELIVERABLE_ID.eq(deliverableId))
```

Comment:
This isn't strictly needed here since there's only one row in the table anyway, but it demonstrates the `where` argument and also maintains the behavior of the old version that filtered by deliverable ID.